wordpress-mobile/WordPress-Android
# Stats: Remove Latest Post Summary card

This PR removes Latest Post Summary card from the defaults on Insights tab on Stats screen. This also pulls the Today card to the top of the list. 

The corresponding [FluxC PR is here](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2297).  Need to update with FluxC trunk version again once it is merged.

Fixes #16010 

| Before | After |
|---|---|
|![Screenshot_20220228_111047](https://user-images.githubusercontent.com/990349/155909485-84eff1ff-84a3-4dd9-9c35-67c7da0dedde.png)|![Screenshot_20220228_105824](https://user-images.githubusercontent.com/990349/155909466-3e8755ed-9dc8-4968-8d93-62802abb5e86.png)|

To test:

Test 1

- Launch app, then click on Stats either through quick links or Stats option below on My Site
- Make sure, it is on Insights tab.  If not on Insights tab then switch to Insights tab
- Notice that there's no **Latest Post Summary card**
- For completeness, compare this version with another version to see Latest Post Summary appears as in Before image above


Test 2

- Launch app, then click on Stats either through quick links or Stats option below on My Site
- Make sure, it is on Insights tab.  If not on Insights tab then switch to Insights tab
- Notice that there's no **Latest Post Summary card**
- Click on ⚙️ icon at the top right hand corner  or + Add new stats card to go to Insights management screen
- Select Latest Post Summary and click SAVE, and return to Insights
- Make sure Latest Post Summary card appears.  Scroll down if necessary to see the card


## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual and unit testing

3. What automated tests I added (or what prevented me from doing so)
Update existing unit tests

PR submission checklist:
